ldms/src/ldmsd/ldmsd_updtr.c

```diff
@@ -575,7 +593,8 @@ void __ldmsd_prdset_lookup_cb(ldms_t xprt, enum ldms_lookup_status status,
 			goto out;
 	}
 	prd_set->state = LDMSD_PRDCR_SET_STATE_READY;
-	ldmsd_log(LDMSD_LINFO, "Set %s is ready\n", prd_set->inst_name);
+	ldmsd_log(LDMSD_LINFO, "Set %s is ready from %s\n", prd_set->inst_name,
```

Comment:
It would be nice to normalize these messages. maybe "prdcr <name>: blah" for all of them instead of switching back and forth between whether the prdcr name comes before or after the set name.

Reply:
I appended the detail so that anyone who already has an expression match against the prior string in some log filter will continue to match.